fastlane/lib/fastlane/actions/reset_simulators.rb

```diff
@@ -2,7 +2,17 @@ module Fastlane
   module Actions
     class ResetSimulatorsAction < Action
       def self.run(params)
-        FastlaneCore::Simulator.reset_all
+        if params[:ios]
```

Comment:
could we potentially make it part of `Simulator` class `device_manager.rb`?
The functionality seems to be reusable and can sit in the core.
There could be another function that reuses `def reset(udid: nil, name: nil, os_version: nil)` under the hood and uses `os_version` parameter.